sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestSettings.cs

```diff
@@ -143,6 +144,17 @@ private static void OverrideFromEnvVariables()
             {
                 Environment.SetEnvironmentVariable(s_digitalTwinHostnameEnv, Instance.DigitalTwinsInstanceHostName);
             }
+
+            string testMode = Environment.GetEnvironmentVariable(s_digitalTwinTestMode);
```

Comment:
This will be set in the test pipeline to Live in tests.yml. For local and PRs, this will set in config file. Locally user config will be used and in PR pipeline, common.config will be used (set to playback)